aspnetcore/tutorials/index.md

```diff
@@ -13,13 +13,48 @@ ms.prod: aspnet-core
 ---
 # Tutorials
 
-- [Your First ASP.NET Core Application on a Mac Using Visual Studio Code](your-first-mac-aspnet.md)
-- [Building your first ASP.NET Core MVC app with Visual Studio](first-mvc-app/index.md)
-- [Building Your First Web API with ASP.NET Core MVC and Visual Studio](first-web-api.md)
-- [Getting started with ASP.NET Core and Entity Framework Core using Visual Studio](../data/ef-mvc/index.md)
-- [Deploy an ASP.NET Core web app to Azure using Visual Studio](publish-to-azure-webapp-using-vs.md)
-- [ASP.NET Core on Nano Server](nano-server.md)
-- [ASP.NET Core and Azure Service Fabric](https://azure.microsoft.com/documentation/articles/service-fabric-add-a-web-frontend/.md)
-- [Creating Backend Services for Native Mobile Applications](../mobile/native-mobile-backend.md)
-- [Developing ASP.NET Core applications using dotnet watch](dotnet-watch.md)
-- [ASP.NET Web API Help Pages using Swagger](web-api-help-pages-using-swagger.md)
+The following step-by-step guides for developing ASP.NET Core applications are available:
+
+## Building basic websites and web applications
```

Comment:
@danroth27  please review my comment, which I'm sure you'll disagree with.  I think this should be the last bullet  as most new folks will want to build a UI with MVC or a Web API.

Other than that, I think this is excellent.